src/Symfony/Bridge/Monolog/Handler/MailerHandler.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Monolog\Handler;
+
+use Monolog\Formatter\FormatterInterface;
+use Monolog\Formatter\HtmlFormatter;
+use Monolog\Formatter\LineFormatter;
+use Monolog\Handler\MailHandler;
+use Monolog\Logger;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+
+/**
+ * @author Alexander Borisov <boshurik@gmail.com>
+ */
+class MailerHandler extends MailHandler
+{
+    /**
+     * @var MailerInterface
+     */
+    private $mailer;
+
+    /**
+     * @var callable|Email
```

Comment:
this should rather be added as phpdoc on the constructor, to describe the type of the public argument rather than the type of the private property.